### DISCUSSION TITLE
installation on Apache or Nginx

### DISCUSSION THREAD

**Discussion by mknotzer** [unoplatform/uno#6661] [Q&A]
web.config works only with IIS. It's crucial to deliver the compressed _compressed_br or _compressed_br files. Where can I find a .htaccess file for Apache resp. a config file for Nginx?

**Comment by jeromelaban** [comment]
web.config is optional, and if you remove this file (or use [WasmShellCompressionLayoutMode](https://github.com/unoplatform/Uno.Wasm.Bootstrap#pre-compression-support)), the tooling will generate in-place `.br` files that can be picked by that web servers (e.g. [Azure Static WebApps how-to](https://platform.uno/docs/articles/guides/azure-static-webapps.html?q=static)).

We do not have documentation available to handle this specific pre-compression case in apache or nginx, but if you do determine the steps to do so, would you be able to make a contribution to [this document](https://github.com/unoplatform/uno/blob/release/stable/3.9/doc/articles/how-to-host-a-webassembly-app.md) ? Thanks!
